### ISSUE TITLE
REQ: Prebuilt Compiler Option

### ISSUE THREAD

**Issue by SmoochySix4** [rdp/ffmpeg-windows-build-helpers#722]
add an option to use an already installed/built version of the mingw32 compiler

**Comment by rdp** [comment]
There "kind of" is in the quick_build folder, give it a shot, it's a
simplified build.

**Comment by SmoochySix4** [comment]
That's awesome. Is there a way to set the main script, easily, to use the already installed mingw32?

**Comment by rdp** [comment]
No, unfortunately.  The already installed mingw32 (at least from packages)
come with some funky options of "static and dynamic" libc++ which might
require some changes...

**Comment by SmoochySix4** [comment]
Thanks for the help. For some reason, there is a lag when running the resulting binary from the main script, but when I used the most recent mingw32 build from Zeranoe's Git, ffmpeg was faster.
